src/lib/get-topper-settings.js

```diff
@@ -10,8 +10,10 @@ const isNews = (content) =>
 	content.annotations &&
 	content.annotations.some((annotation) => annotation.prefLabel === 'News');
 
+const isLiveBlogV2 = (content) => content.type === 'live-blog-package';
+
 const isLiveBlogOrPackage = (content) =>
```

Comment:
This method might be better named `isLiveBlogV1OrPackage()` to clarify that it isn't identifying the new live blogs.